<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSliderMin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>36</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderMax">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>36</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberMax">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderTime">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing(s)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonDisconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEditIP"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>221</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
